DependencyInjection/Compiler/FilterPass.php

```diff
@@ -30,47 +28,31 @@ class FilterPass implements CompilerPassInterface
      */
     public function process(ContainerBuilder $container)
     {
-        foreach ($container->findTaggedServiceIds('es.filter_manager') as $managerId => $managerTags) {
-            $managerDefinition = $container->getDefinition($managerId);
-            
-            foreach ($container->findTaggedServiceIds('ongr_filter_manager.filter') as $filterId => $filterTags) {
-                foreach ($filterTags as $tag) {
-                    if (array_key_exists('manager', $tag)
-                        && $managerId === ONGRFilterManagerExtension::getFilterManagerId($tag['manager'])
-                    ) {
-                        if (!array_key_exists('filter_name', $tag)) {
-                            throw new InvalidConfigurationException(
-                                sprintf('Filter tagged with `%s` must have `filter_name` set.', $filterId)
-                            );
-                        }
+        foreach ($container->findTaggedServiceIds('ongr_filter_manager.filter') as $filterId => $filterTags) {
+            $tag = array_shift($filterTags);
```

Comment:
What if for some reason developer needs to declare same service as multiple filters? We should iterate trough all tags here.